timelock-impl/src/test/java/com/palantir/atlasdb/timelock/lock/watch/LockWatchingServiceImplTest.java

```diff
@@ -104,7 +108,7 @@ public void runTaskRunsExclusivelyOnLockLog() throws InterruptedException {
         Set<LockDescriptor> locks = ImmutableSet.of(CELL_DESCRIPTOR);
 
         Future<?> registerLock = executor.submit(() -> {
-            lockWatcher.registerLock(locks, TOKEN);
+            lockWatcher.registerLock(locks, TOKEN, Optional.empty());
```

Comment:
Any reasons why this is `lockWatcher.registerLock(locks, TOKEN, Optional.empty());` rather than just `lockWatcher.registerLock(locks, TOKEN);` ?

Reply:
I reran the tests and it is indeed not necessary, the default method is fine